manifests/profile.pp

```diff
@@ -108,6 +114,10 @@
     $group_real = $group
   }
 
+  if ($source_profile != undef and $credential_source != undef) {
+    fail('aws cli profile cannot contain both source_profile and credential_source config option')
```

Comment:
It might be good to have a spec to test this. Do you think you could add one?

Reply:
tests added